src/libraries/System.Drawing.Common/tests/Printing/PrinterSettingsTests.cs

```diff
@@ -431,12 +431,12 @@ public void IsDirectPrintingSupported_ImageNotSupported_ReturnsExpected()
             }
         }
 
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/22221", TestPlatforms.AnyUnix)]
-        [ConditionalFact(Helpers.AnyInstalledPrinters, Helpers.IsDrawingSupported)]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [ConditionalFact(typeof(PrinterSettingsTests), nameof(CanTestSetHdevmode_IntPtr_Success))]
         public void SupportsColor_ReturnsExpected()
         {
-            var printerSettings = new PrinterSettings();
-            bool supportsColor = printerSettings.SupportsColor;
+            var printerSettings = new PrinterSettings() { PrinterName = GetNameOfTestPrinterSuitableForDevModeTesting() };
```

Comment:
`GetNameOfTestPrinterSuitableForDevModeTesting` has a comment that describes that we are looking for a printer that supports duplex, etc, please add that it should support color as well.

Reply:
Also, had you double-checked that `Microsoft XPS Document Writer` supports color?

Reply:
> GetNameOfTestPrinterSuitableForDevModeTesting has a comment that describes that we are looking for a printer that supports duplex, etc, please add that it should support color as well.

Yeah, I'll update the comment.

> Also, had you double-checked that Microsoft XPS Document Writer supports color?

I only found this, which seems like it does.
https://docs.microsoft.com/en-us/windows-hardware/drivers/print/improved-color-printing

That's why I sent it to the PR since we run tests in Windows 7 and Windows 8, to see if the test failed or not. From looking at the console of the Windows 7 run it didn't and it also didn't skip the test at all.

```
C:\h\w\A59E091E\w\A8EC091E\e>"C:\h\w\A59E091E\p\dotnet.exe" exec --runtimeconfig System.Drawing.Common.Tests.runtimeconfig.json --depsfile System.Drawing.Common.Tests.deps.json xunit.console.dll System.Drawing.Common.Tests.dll -xml testResults.xml -nologo -nocolor -notrait category=IgnoreForCI -notrait category=OuterLoop -notrait category=failing -notrait category=nonnetcoreapptests -notrait category=nonwindowstests  
  Discovering: System.Drawing.Common.Tests (method display = ClassAndMethod, method display options = None)
  Discovered:  System.Drawing.Common.Tests (found 1930 of 1965 test cases)
  Starting:    System.Drawing.Common.Tests (parallel test collections = on, max threads = 2)
    MonoTests.System.Drawing.Imaging.JpegCodecTest.Bitmap8bbpIndexedGreyscalePixels [SKIP]
      Condition(s) not met: "GetGdiPlusIsAvailableNotWindows7"
    MonoTests.System.Drawing.Imaging.JpegCodecTest.Bitmap8bbpIndexedGreyscaleData [SKIP]
      Condition(s) not met: "IsNotWindows7"
    System.Drawing.Tests.GdiplusTests.IsAtLeastLibgdiplus6 [SKIP]
      Condition(s) not met: "IsOSX"
  Finished:    System.Drawing.Common.Tests
=== TEST EXECUTION SUMMARY ===
   System.Drawing.Common.Tests  Total: 4671, Errors: 0, Failed: 0, Skipped: 3, Time: 4.901s
```

Reply:
>I only found this, which seems like it does.
docs.microsoft.com/en-us/windows-hardware/drivers/print/improved-color-printing

Great! Consider adding this link in a comment.

>That's why I sent it to the PR since we run tests in Windows 7 and Windows 8, to see if the test failed or not. From looking at the console of the Windows 7 run it didn't and it also didn't skip the test at all.

Cool!